doc/connectivity/region_extraction.rst

```diff
@@ -34,50 +34,50 @@ which is already preprocessed and publicly available at
 datasets.
 
 
-.. literalinclude:: ../../examples/03_connectivity/plot_extract_regions_canica_maps.py
+.. literalinclude:: ../../examples/03_connectivity/plot_extract_regions_dictlearn_maps.py
```

Comment:
The object is called DictLearning, so I would rather call this example plot_extract_regions_dictlearning_maps.py

Sorry for the annoying comment that comes in a bit late.